apps/docs/src/guide/transactions/snippets/assemble-tx/change-output-collision.ts

```diff
@@ -31,3 +31,14 @@ const accountCoinQuantities: AccountCoinQuantity[] = [
 // #endregion change-output-collision
 
 console.log('accountCoinQuantities', accountCoinQuantities);
+
+const promise = provider.assembleTx({
+  request,
+  feePayerAccount: accountA,
+  accountCoinQuantities,
+});
+
+expect(promise).rejects.toThrow();
+expect(promise).rejects.toMatchObject({
+  code: 'change-output-collision',
+});
```

Comment:
Should we use test asserts within doc snippets?

Reply:
No, we should not.